### DISCUSSION TITLE
能不能支持多账号或者多开软件

### DISCUSSION THREAD

**Discussion by zhenge666** [Mrs4s/go-cqhttp#2210] [General]
想开几个账号同时运行，以后能不能支持，现在多开也不支持，有一个运行第二个直接报错，不知道多开软件行不行。

**Comment by zhx79513** [comment]
复制两个go-cqhttp到不同目录下，然后配置不一样的端口和QQ都不行吗？

**Comment by xiangxiangxiong9** [comment]
提示什么报错呢...

**Comment by zhenge666** [comment]
我设置错了，一个开了http，另一个打开启动不起来是端口冲突了，关闭http可以多个文件夹多开
